### PR TITLE
Bump to v3.0.0-rc.3

### PR DESCRIPTION
**Pull request by Connor1996**
## What have you changed? (mandatory)
Bump to v3.0.0-rc.3

## What are the type of the changes? (mandatory)
- Misc (other changes)

### CODE REVIEW

**Comment by Connor1996** [comment]
PTAL @dcalvin

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,11 +4,38 @@ See also [TiDB Changelog](https://github.com/pingcap/tidb/blob/master/CHANGELOG.
 
 ## [Unreleased]
 
+## [3.0.0-rc.3]
+
++ Engine
+    - Check iterator status when scanning. [4936](https://github.com/tikv/tikv/pull/4936)
+    - Fix ingested file and direcotry not being sync. [4937](https://github.com/tikv/tikv/pull/4937)
```
Comment (dcalvin) [inline CHANGELOG.md]:
```suggestion
    - Fix the issue that ingested files and directory are not synchronized. [4937](https://github.com/tikv/tikv/pull/4937)
```

**Pull request merged**